server/service/schema/transactions.schema.js

```diff
@@ -0,0 +1,67 @@
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+
+const transaction = new Schema(
+  {
+    date: {
+      // TODO co przychodzi z kompnentu?
+      //type: String,
+    },
+    year: {
+      type: Number,
+    },
+    month: {
+      type: String,
+      enum: [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December',
+      ],
+    },
+    type: {
+      type: String,
+      enum: ['income', 'expense'],
+      required: [true, 'Type is required'],
+    },
+    category: {
+      type: String,
+      enum: [
+        'Main expenses',
+        'Products',
+        'Car',
+        'Self care',
+        'Child care',
+        'Household products',
+        'Education',
+        'Leisure',
+        'Other expenses',
+        'Entertainment',
+      ],
+    },
+    comment: {
+      type: String,
+    },
+    sum: {
+      type: Number,
+    },
+    owner: {
```

Comment:
Pole owner jest wymaganym elementem schematu: require: [true, 'owner is required'] :)